modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/Oauth2PersistenceProcessorTestCase.java

```diff
@@ -72,41 +80,45 @@ public void testEncryptionDecryptionPersistenceProcessor() throws Exception {
         changeISConfiguration();
         super.init(TestUserMode.SUPER_TENANT_USER);
 
-        OAuthConsumerAppDTO oAuthConsumerAppDTO = createApplication("app2");
-        Assert.assertNotNull(oAuthConsumerAppDTO, "Application creation failed.");
+        ApplicationResponseModel application = createApplication("app2");
+        Assert.assertNotNull(application, "Application creation failed.");
+        String applicationId2 = application.getId();
 
-        consumerKey2 = oAuthConsumerAppDTO.getOauthConsumerKey();
+        OpenIDConnectConfiguration oidcConfig = getOIDCInboundDetailsOfApplication(applicationId2);
+        String consumerKey2 = oidcConfig.getClientId();
         Assert.assertNotNull(consumerKey2, "Application creation failed.");
 
-        String consumerSecret2 = oAuthConsumerAppDTO.getOauthConsumerSecret();
+        String consumerSecret2  = oidcConfig.getClientSecret();
```

Comment:
```suggestion
        String consumerSecret2 = oidcConfig.getClientSecret();
```

Reply:
Fixed with https://github.com/wso2/product-is/pull/16271/commits/e9343598362e5021e1f6832b61ef1997495afeda